packages/permissionless/accounts/index.ts

```diff
@@ -85,5 +91,8 @@ export {
     type PrivateKeyToSafeSmartAccountParameters,
     type SignerToEcdsaKernelSmartAccountParameters,
     type SignerToBiconomySmartAccountParameters,
-    type PrivateKeyToBiconomySmartAccountParameters
+    type PrivateKeyToBiconomySmartAccountParameters,
+    type EtherspotSmartAccount,
+    type SignerToEtherspotSmartAccountParameters,
```

Comment:
can we also add `PrivateKeyToEtherspotSmartAccount` file

Reply:
I have added **privateKeyToEtherspotSmartAccount** file.